packages/modules/src/modules/banners/choiceCardsBanner/components/SupportCta.tsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import { addRegionIdAndTrackingParamsToSupportUrl } from '@sdc/shared/dist/lib';
+import { Tracking } from '@sdc/shared/dist/types';
+import { neutral, space } from '@guardian/src-foundations';
+import { css } from '@emotion/react';
+import { BannerEnrichedCta } from '../../common/types';
+import { Hide } from '@guardian/src-layout';
+import { Button } from './Button';
+import { ChoiceCardSelection } from '../ChoiceCardsBanner';
+
+const buttonOverrides = css`
+    margin-right: ${space[3]}px;
+    margin-bottom: ${space[3]}px;
+    background: ${neutral[0]};
+    color: ${neutral[100]} !important;
+
+    &:hover {
+        background-color: rgba(0, 0, 0, 0.75);
```

Comment:
Have'nt seen this used before, opacity presume 75% is the standard guardian design rules?

Reply:
This isn't part of the guardian design system, these are all css overrides